### PR TITLE
Setup Travis CI 

### LINKED ISSUE #45: Set up Travis CI for continuous integration

### PR DESCRIPTION
**Pull request by nicolegooden**
### Participating Group Members:

- Nicole Gooden

### Code Highlights:

- Replaced unit test for `HelloWorld` with unit test for `TheHeader` in the example test file
- Created and edited the `.travis.yml` file for Travis CI

### Have Tests Been Added?

- [ ] No.
- [ ] Yes, but all tests are not passing.
- [X] Yes, and all are passing.

### Any background context you want to provide?

Setting this up for the first time in Vue, rather than React, was tricky.  We have a successful build, though.  Planning to work on deployment to Heroku tomorrow.

### Message/Questions for reviewer:

N/A

### Issues:

- Closes #45

### Screenshots (if appropriate):

Screen Shot 2021-01-05 at 7 27 03 PM

### Tracking Consistency:

- [X] added appropriate labels
- [X] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas - N/A
- [X] All new and existing tests passed

- [X] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
This will help us out a lot as we move forward with the project. Thank you for taking the lead with this and getting CI setup!

**Review by stacyp2006 — APPROVED** [review]
Thanks for taking this on Nicole!

File: tests/unit/example.spec.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-  it('renders props.msg when passed', () => {
-    const msg = 'new message'
-    const wrapper = shallowMount(HelloWorld, {
-      propsData: { msg }
-    })
-    expect(wrapper.text()).toMatch(msg)
+describe('TheHeader', () => {
+  it('renders TheHeader component', () => {
+    const header = mount(TheHeader)
+    expect(header.exists()).toBe(true)
```
Comment (bretm9) [inline tests/unit/example.spec.js:7]: Thanks for this simple test! Great to know our test suite is up and running.

**Review by bretm9 — APPROVED** [review]
Travis CI will keep us in-check as we begin to test in the upcoming days. Thanks for implementing this. Approved, and merging now.

**Pull request merged**